packages/kit/src/vite/index.js

```diff
@@ -388,6 +389,8 @@ function kit() {
 		 * @see https://vitejs.dev/guide/api-plugin.html#configureserver
 		 */
 		async configureServer(vite) {
+			const configWarning = create_overridden_config_action(original_config, vite_config);
+			add_post_listening_actions(vite, configWarning);
```

Comment:
```suggestion
			const config_warning = create_overridden_config_action(original_config, vite_config);
			add_post_listening_actions(vite, config_warning);
```

I think this needs a snake case.

Reply:
I am physically unable to write JavaScript variables in snake case 😂